File: Src/App/zabotalaboratory/zabotalaboratory.Analyses.Database/Entities/AnalysesResult.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace zabotalaboratory.Analyses.Database.Entities
{
    public class AnalysesResult
    {
        public virtual int Id { get; set; }

        public virtual string Result { get; set; }

        [Required]
        [ForeignKey(nameof(LaboratoryAnalysesTests))]
        public virtual int LaboratoryAnalysesTestsId { get; set; }

        [Required]
        public virtual LaboratoryAnalysesTests LaboratoryAnalysesTests { get; set; }
    }
}

File: Src/App/zabotalaboratory_client/src/app/shared/models/analyses/analyses-result.ts
import {LaboratoryAnalysesTests} from "./laboratory-analyses-tests";

export class AnalysesResult {
  public id: number;
  public result: string;
  public laboratoryAnalysesTest: LaboratoryAnalysesTests;
  public units: string;
  public referenceLimits: string;
}
